src/eval-frame/actions/actions.js

```diff
@@ -60,35 +60,41 @@ export function setConsoleLanguage(language) {
   };
 }
 
-export function appendToEvalHistory(
-  cellId,
+export function addToConsole({
+  historyType,
   content,
   value,
-  historyOptions = {}
-) {
-  const historyType =
-    historyOptions.historyType === undefined
-      ? "CELL_EVAL_VALUE"
-      : historyOptions.historyType;
-
+  level,
+  language,
+  historyId = generateRandomId()
+}) {
   const historyAction = createHistoryItem({
     content,
     historyType,
-    historyId: historyOptions.historyId,
-    lastRan: historyOptions.lastRan
+    historyId,
+    level,
+    language
   });
-  historyAction.type = "APPEND_TO_EVAL_HISTORY";
+  historyAction.type = "ADD_TO_CONSOLE";
```

Comment:
should this be `"ADD_TO_CONSOLE_HISTORY"` for sake of clarity?

Reply:
sure that sounds good

Reply:
looking through the code a little more, i think this is a great idea actually, seems substantially clearer. likewise for the action creator `addToConsole` -> `addToConsoleHistory` above